vendor/bat-native-ads/src/bat/ads/internal/covariates/covariate_manager.h

```diff
@@ -10,6 +10,8 @@
 
 #include "base/containers/flat_map.h"
```

Comment:
not used

Reply:
`covariate_log_entries_` is a `flat_map`